pallets/file-system/src/utils.rs

```diff
@@ -400,7 +400,6 @@ where
 
         if response == BucketMoveRequestResponse::Rejected {
             <PendingBucketsToMove<T>>::remove(&bucket_id);
```

Comment:
you could also just call this once since we delete it whether the msp rejects or accepts the request.

Reply:
Agreed

Reply:
I refactored this! Also the MoveBucketAccepted event was being emitted twice so I removed the call from `utils.rs`